src/SFA.DAS.EAS.Account.Api.UnitTests/SFA.DAS.EAS.Account.Api.UnitTests.csproj

```diff
@@ -226,6 +227,14 @@
       <Project>{40190475-1220-4cf8-8a60-223facafeda1}</Project>
       <Name>SFA.DAS.EAS.Account.Api</Name>
     </ProjectReference>
+    <ProjectReference Include="..\SFA.DAS.EmployerAccounts.Api.Client\SFA.DAS.EmployerAccounts.Api.Client.csproj">
```

Comment:
I think we've talked about this before and I may have changed my mind! 😁 Can the reference be done as a nuget package?  I know it's a pain but it means we'll be able to lift and shift accounts into its own repo.

Reply:
Yup, it's going to have to be a nuget reference. Might be worthwhile changing it over closer to the repo move though, to ease development in the meantime?

Reply:
We're getting very close to moving the repo, I'm hopeful we'll be there in the next sprint.

Reply:
The PR no longer references the client, and references to the types project have been converted to nuget references.